recipes/oslo.concurrency/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "oslo.concurrency" %}
+{% set version = "3.29.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d5f5ccca934a7e736b934044e1a347da2e139db6e4df3a2ac5dedc15e5287c81
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pbr!=2.1.0,>=2.0.0
+    - pip
+    - python
+  run:
+    - pbr!=2.1.0,>=2.0.0
+    - oslo.config>=5.2.0
+    - oslo.i18n>=3.15.3
+    - oslo.utils>=3.33.0
+    - six>=1.10.0
+    - fasteners>=0.7.0
+    - enum34>=1.0.4  # [py2k]
```

Comment:
There needs to be a single space between the package name and the pin, e.g. `pbr !=2.1.0,>=2.0.0`